fe/fe-core/src/main/java/com/starrocks/metric/MetricRepo.java

```diff
@@ -813,6 +831,16 @@ private static void collectDatabaseMetrics(MetricVisitor visitor) {
             tableNum.setValue(db.getTableNumber());
             tableNum.addLabel(new MetricLabel("db_name", dbName));
             visitor.visit(tableNum);
+
+            GaugeMetric<Long> dbSizeBytesTotal = new GaugeMetric<Long>("db_size_bytes",
+                    MetricUnit.BYTES, "total size of db in bytes") {
+                @Override
+                public Long getValue() {
+                    return db.getUsedDataQuotaWithLock();
+                }
+            };
+            dbSizeBytesTotal.addLabel(new MetricLabel("db_name", dbName));
```

Comment:
this api call can be very slow

Reply:
How will this line slow down the API call? Do you have any advice on other API methods I can use for retrieving the database size?